packages/liveblocks-react/src/types/index.ts

```diff
@@ -159,6 +159,12 @@ export type InboxNotificationsQuery = {
    * all the user's inbox notifications will be returned.
    */
   roomId?: string;
+
+  /**
+   * Whether to only return inbox notifications for a specific kind. If not provided,
+   * all the user's inbox notifications will be returned.
```

Comment:
"all the user's inbox notifications will be returned" sounds like `roomId` would be ignored to me, maybe "all inbox notification kinds will be returned"?